### PR TITLE
[patch]: Update documentation

### PR DESCRIPTION
**Pull request by gavindidrichsen**
Signed-off-by: Gavin Didrichsen <gavin.didrichsen@gmail.com>

### CODE REVIEW

File: controls/patch_exists.rb [hunk 1]
```diff
@@ -7,16 +7,17 @@
...
   describe plan_installation_directory do
     its('exit_status') { should eq 0 }
     its('stdout') { should_not be_empty }
+    its('stderr') { should be_empty }
   end
 
-  command_relative_path = input('command_relative_path', value: '/bin/patch')
-  command_full_path = plan_installation_directory.stdout.strip + "#{command_relative_path}"
+  command_relative_path = input('command_relative_path', value: 'bin/patch')
+  command_full_path = File.join(plan_installation_directory.stdout.strip, "#{command_relative_path}")
```
Comment (skpaterson) [inline controls/patch_exists.rb]: This line can be simplified slightly:
``` 
 command_full_path = File.join(plan_installation_directory.stdout.strip, command_relative_path)
```

File: controls/patch_works.rb [hunk 2]
```diff
@@ -7,19 +7,22 @@
...
     its('exit_status') { should eq 0 }
     its('stdout') { should_not be_empty }
+    its('stderr') { should be_empty }
   end
   
-  plan_pkg_ident = ((plan_installation_directory.stdout.strip).match /(?<=pkgs\/)(.*)/)[1]
-  plan_pkg_version = (plan_pkg_ident.match /^#{plan_origin}\/#{plan_name}\/(?<version>.*)\//)[:version]
-  describe command("DEBUG=true; hab pkg exec #{plan_pkg_ident} patch --version") do
+  command_relative_path = input('command_relative_path', value: 'bin/patch')
+  command_full_path = File.join(plan_installation_directory.stdout.strip, "#{command_relative_path}")
```
Comment (skpaterson) [inline controls/patch_works.rb]: Same as above I think.

File: controls/patch_exists.rb (hunk 1 above)
Comment (gavindidrichsen) [reply controls/patch_exists.rb]: Done.

File: controls/patch_works.rb (hunk 2 above)
Comment (gavindidrichsen) [reply controls/patch_works.rb]: Done.

**Review by skpaterson — APPROVED** [review]
Thanks @gavindidrichsen

**Pull request merged**